examples/notebooks/create_deployment.ipynb

```diff
@@ -12,48 +23,288 @@
    ]
   },
   {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Load config from default location."
+   ]
+  },
+  {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 2,
    "metadata": {
     "collapsed": true
    },
    "outputs": [],
    "source": [
-    "client.Configuration().host=\"http://localhost:8080\""
+    "config.load_kube_config()\n",
+    "extensions = client.ExtensionsV1beta1Api()"
```

Comment:
@pokoli do you mind if we merge this PR and close yours. This addresses more issues in the notebook. Once merged we can discuss more.

The general issue with these notebooks is to figure out a way to quickly test them.

I am tempted to add two steps in the notebooks with a try loop that loads in cluster and loads out of cluster.